rts/lra/lra-proxy/api/src/main/java/io/narayana/lra/client/participant/TerminationException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2017, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package io.narayana.lra.client.participant;
+
+public class TerminationException extends Throwable {
```

Comment:
would not be good to add javadoc with explanation what is this exception used for? Plus I don't think it's a good idea to extend directly from `Throwable`.

Reply:
Again the javadoc is in the interface LRAParticipant where it is used.

And, okay I will extend from Exception instead.

Reply:
@mmusgrov ah, I can see now, I was asking the same question some days ago. ay... Btw. what to use in the class implementation something like

```
/**
 * {@inheritDoc} 
**/
```

Reply:
I've not come across that before - I will use it